src/AFL-3.0.xml

```diff
@@ -0,0 +1,208 @@
+<spdx name="Academic Free License v3.0" identifier="AFL-3.0" osi-approved="true">
+  <urls>
+    <url>http://www.rosenlaw.com/AFL3.0.htm</url>
```

Comment:
> … we might want to keep them for archival purposes, and just add new URLs as new authoritative(-ish) URLs come up.

We have Git for archival ;).  If we want to keep known-dead URLs in the XML, I recommend we at least mark them up as such (e.g. `<url dead="true">…</url>`).

> Since this affects a number of licenses, can you open an issue where we can decide how to handle linkrot (or document it, if there's already a policy), before any more PRs?

I haven't been able to turn up a policy around this.  But it seems like an issue for the legal team, so I don't know if an issue in this repo makes sense.  Maybe I should mail spdx-legal?  Or file an issue in sdpx/spdx-spec?  Or maybe this repo is the right place?